components/nexus-core/src/main/java/org/sonatype/nexus/internal/web/ErrorPageServlet.java

```diff
@@ -98,19 +98,23 @@ public ErrorPageServlet(final Provider<SystemStatus> systemStatus,
     this.webUtils = checkNotNull(webUtils);
   }
 
+  @SuppressWarnings("deprecation")
   @Override
   protected void service(final HttpServletRequest request, final HttpServletResponse response)
       throws ServletException, IOException
   {
     webUtils.addNoCacheResponseHeaders(response);
 
+    @SuppressWarnings("unused")
     String servletName = (String) request.getAttribute(ERROR_SERVLET_NAME);
+    @SuppressWarnings("unused")
     String requestUri = (String) request.getAttribute(ERROR_REQUEST_URI);
     Integer errorCode = (Integer) request.getAttribute(ERROR_STATUS_CODE);
     String errorMessage = (String) request.getAttribute(ERROR_MESSAGE);
-    Class causeType = (Class) request.getAttribute(ERROR_EXCEPTION_TYPE);
+    @SuppressWarnings("unused")
+    Class<?> causeType = (Class<?>) request.getAttribute(ERROR_EXCEPTION_TYPE);
+    @SuppressWarnings("unused")
     Throwable cause = (Throwable) request.getAttribute(ERROR_EXCEPTION);
```

Comment:
Commenting out those lines causes the "unused" warnings to migrate to the now unused constant string. 6 of one, half-dozen of another...